sis-local/example/sis_app.py

```diff
@@ -0,0 +1,57 @@
+from snowflake.snowpark.context import get_active_session
+from snowflake.snowpark.exceptions import SnowparkSessionException
+
+
+def in_snowpark() -> bool:
+    try:
+        sess = get_active_session()
+        return True
+    except SnowparkSessionException:
+        return False
+
+
+if in_snowpark():
+    import streamlit as st
+    session = get_active_session()
+else:
+    import streamlit_in_snowflake as st
+    session = st.LocalSnowparkConnection().connect()
```

Comment:
I'd be a little concerned about this misleading people to think that LocalSnowparkConnection is part of streamlit in general. I wonder about separating them out a bit in the package, and doing:

```python
import streamlit_in_snowflake as st
from streamlit_in_snowflake.local_connection import get_active_session
session = get_active_session()
```

WDYT?

Reply:
Yep this makes sense. WDYT about the new version? I didn't want to confuse the local method with the snowpark API since it does behave a little differently. Again I don't think this part will be needed long term.

```python
if running_in_sis():
    import streamlit as st
    session = get_active_session()
else:
    import streamlit_in_snowflake as st
    from streamlit_in_snowflake.local_session import get_local_session
    session = get_local_session()
```